README.md

```diff
@@ -20,7 +20,7 @@
   - [fund](#fund)
   - [nuke](#nuke)
   - [print](#print)
-  - [simulate](#simulate)
+  - [simulate](#simulate) *(deprecated)*
```

Comment:
This link doesn't work. Maybe just move `deprecated` from heading. And put somewhere in description.